client/libs/theme/variables.scss

```diff
@@ -0,0 +1,7 @@
+$primary: #34587C;
+$accent: #83B6EF;
+$other-accent: #FB8360;
```

Comment:
Can we rename this?

Reply:
![image](https://user-images.githubusercontent.com/38518209/68981087-8f25f980-07bf-11ea-830e-ccd1025bc05c.png)
here is an image showing the updated theme works